pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -226,11 +226,12 @@ type OpenstackClusterProviderStatus struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// TODO: create and provide this info later
```

Comment:
I think we should remove this entirely. Maybe this came only in due to an copy and paste error early on. And I am very opinionated in not having anything secret (CAPrivateKey) stored outside of a secret.